### ISSUE TITLE
Is there support for automatically inserting a snippet when a certain string is entered?

### ISSUE THREAD

**Issue by sneer-chase-scouting-descent** [ms-jpq/coq_nvim#342]
What I mean by the title is, for example: Consider that I have a snippet set for inserting `$${1}$${0}` that I would use for inline math in markdown. Is there a way for me to set it up so that I can just type `hv` and the snippet would automatically be triggered? See <https://castel.dev/post/lecture-notes-1/> for more examples (this is what my current setup in vscodium is based on but I would like to use coq.nvim and nvim).
Thanks for the plugin btw!

**Comment by ms-jpq** [comment]
no, hummmm idk if i should add something for it, since it's like kinda weird, interesting idea tho
